setup.py

```diff
@@ -34,6 +34,10 @@ def extract_version():
 def read(*parts):
     return open(os.path.join(rootpath, *parts), 'r').read()
 
+
+pkg_data = {'': ['templates/*.tpl']}
```

Comment:
@rsignell-usgs ideally we would have one template for each model/type of aggregation.